Tools/TestWebKitAPI/Tests/ios/DragAndDropTestsIOS.mm

```diff
@@ -1211,7 +1211,7 @@ static BOOL overrideIsInHardwareKeyboardMode()
 
         auto simulator = adoptNS([[DragAndDropSimulator alloc] initWithWebView:webView.get()]);
         [simulator setExternalItemProviders:@[ itemProvider.get() ]];
-        [simulator runFrom:CGPointMake(50, 50) to:CGPointMake(150, 50)];
+        [simulator runFromElement:@"output" toElement:@"droparea"];
```

Comment:
Hm...since these are element IDs and the method takes a query selector now, I would expect that these need to be `@"#output"` and `@"#droparea"` respectively? (I had suggested making the helper take a query selector instead of an ID, so that it could be used for scenarios where you wanted to start or end drags on elements that didn't have an `id` attribute, e.g. the only `img` element in the document or something along those lines).

Reply:
They were being used as selectors

```js
let element = document.querySelector('#%@');
```

_but_ of course this is not apparent at the call-site, which is confusing!

Instead I've renamed all parameters to indicate that we're expecting selector queries and updated the call-site to read `@"#output"` and `@"#droparea"` respectively.